include/flatbuffers/idl.h

```diff
@@ -296,6 +296,7 @@ struct FieldDef : public Definition {
         shared(false),
         native_inline(false),
         flexbuffer(false),
+        nullable(true),
```

Comment:
this should probably be `false`?

Reply:
Maybe... sure. I could go either way on this one. I set it to true because everything except scalars are nullable.